lib/koans/09_pattern_matching.ex

```diff
@@ -5,6 +5,16 @@ defmodule PatternMatching do
     assert match?(1, 1)
   end
 
+  koan "a pattern can change" do
```

Comment:
Don't really see where the blanks would go in these two. Unfortunately I don't have a better suggestion currently either...